docs/index.md

```diff
@@ -8,44 +8,50 @@
+Most data science solutions and platforms on the market today begin and therefore emphasize the research workflow. 
+When it comes time to integrate the generated models into real-world AI applications, they have significant functionality gaps.
 
-After you develop your model and feature engineering logic you need to deploy those into production pipelines 
-with real-time feature engineering, online model serving, API and data integrations, model and data quality 
-monitoring, no intrusive upgrades and so on. 
+These types of solutions tend to be useful only for the model development flow, and contribute very little to the production pipeline: 
+automated data collection and preparation, automated training and evaluation pipelines, real-time application pipelines, 
+data quality and model monitoring, feedback loops, etc.
 
-However, the implementation is challenging: various environments ("runtimes") use different configurations, parameters, and data sources.
-In addition, multiple frameworks and platforms are used to focus on different stages of the development life cycle.
-This leads to constant development and DevOps/MLOps work.
+To address the full ML application lifecycle, it’s common for organizations to combine many different tools which then forces 
+them to develop and maintain complex glue layers, or introduce manual processes, and creates technology silos that slow down 
+developers and data scientists. 
 
-Furthermore, as your project scales, you need greater computation power or GPUs, and you need to access large-scale data sets.
-This cannot succeed on laptops.
-You need a way to seamlessly run your code on a remote cluster and automatically scale it out.
+With this disjointed approach, the ML team must re-engineer the entire flow to fit production environments and methodologies 
+and consumes excessive resources. Organizations need a way to streamline the process, 
+automate as many tasks as possible, and break the silos between data, ML, and DevOps/MLOps teams.
 
 <a id="why-mlrun"></a>
-## Why MLRun?
+## MLRun - The Open Source MLOps Orchestration
 
-When running ML experiments, ideally you can record and version your code, configuration, outputs, and associated inputs (lineage), 
-so you can easily reproduce and explain your results.
-The fact that you probably need to use different types of storage (such as files and AWS S3 buckets) and various databases, further complicates the implementation.
+Instead of this siloed, complex and manual process, MLRun enables production pipeline design using a modular strategy, 
+where the different parts contribute to a continuous, automated, and far simpler path from research and development to scalable 
+production pipelines, without the need to refactor code, add glue logic, and spend significant efforts on data and ML engineering.
```

Comment:
without refactoring code, adding glue logic, or spending significant efforts on data and ML engineering.